python/pyarrow/tests/deserialize_buffer.py

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can you add a comment that this file is called by another test? I was at first confused why this is needed as it does not contain any methods prefixed with `test_` ;)

Reply:
fixed